lib/ship_compliant/add_update_brand.rb

```diff
@@ -37,8 +37,8 @@ class AddUpdateBrand
     #     ShipCompliant::AddUpdateBrand.product({
     #       # brand attributes
     #     }, update_mode: 'UpdateExisting')
-    def self.brand(brand, options = {})
-      result = ShipCompliant.client.call(:add_update_brand, {
+    def self.brand(brand, options: {}, configuration: :default)
```

Comment:
This introduces a breakage but IMO is worth it for having the trailing configuration kw param.

The reason I had to change this is because Ruby can't handle method signatures like this

```ruby
def self.brand(options = {}, configuration:)
```

because it gets confused and thinks the last kw is a part of the options hash.